.ado/jobs/cli-init.yml

```diff
@@ -54,14 +54,14 @@ jobs:
             platform: x64
             projectType: app
             additionalInitArguments:
-            additionalRunArguments:
+            additionalRunArguments: --deploy-from-layout
```

Comment:
So are we no longer testing CLI release with certs?

Reply:
We could if we wanted to in CI! I can do the same process as I did for other release builds and pull in a cert, but we won't have access to this in PR pipelines.